Sources/Public/CalendarView.swift

```diff
@@ -75,9 +75,21 @@ public final class CalendarView: UIView {
 
   // MARK: Public
 
-  /// A closure (that is retained) that is invoked whenever a day is selected.
+  /// A closure (that is retained) that is invoked whenever a day is selected. It is the responsibility of your feature code to decide what to
+  /// do with each day. For example, you might store the most recent day in a selected day property, then read that property in your
+  /// `dayItemProvider` closure to add specific "selected" styling to a particular day view.
   public var daySelectionHandler: ((Day) -> Void)?
 
+  /// A closure (that is retained) that is invoked during a multiple-selection-drag-gesture. Multiple selection is initiated with a long press,
+  /// followed by a drag / pan. As the gesture crosses over more days in the calendar, this handler will be invoked with each new day. It
+  /// is the responsibility of your feature code to decide what to do with this stream of days. For example, you might convert them to
+  /// `Date` instances, and use them as input to the `dayRangeItemProvider`.
```

Comment:
know you love the micro nits
```suggestion
  /// `Date` instances and use them as input to the `dayRangeItemProvider`.
```